<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameConfiguration</class>
 <widget class="QDialog" name="GameConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 20pt &quot;Arial&quot;;
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>Tic-Tac-Toe</string>
   </property>
  </widget>
  <widget class="QLabel" name="player1label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Arial&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Player 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="player2label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Arial&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Player 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="player1line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="player2line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>138</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="twoplayermode">
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Arial&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Two Player mode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="AIMode">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Arial&quot;;
font: 700 10pt &quot;Arial&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>AI mode</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GameConfiguration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GameConfiguration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>player1line</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GameConfiguration</receiver>
   <slot>updateOKButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>player2line</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GameConfiguration</receiver>
   <slot>updateOKButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateOKButton()</slot>
 </slots>
</ui>
